packages/enzyme-context-apollo-3/src/Utils.ts

```diff
@@ -0,0 +1,65 @@
+import { IMocks, IMockFn } from 'graphql-tools';
```

Comment:
Hmmm might wanna consider putting this in a shared `enzyme-context-apollo-utils` package but I think we can punt on that for now and do it later if we need to add more utils. (I don't feel compelled to do it now considering it's only one function.)

Reply:
Agreed, we can revisit if it looks like there's real risk of divergence